doc.go

```diff
@@ -40,75 +37,79 @@
 //
 // You can generate a fingerprint from a media file:
 //
-//     ft, err := client.FingerprintFile("/path/to/file.mp4")
-//     if err != nil {
-//         panic(err)
-//     }
+//	ft, err := client.FingerprintFile("/path/to/file.mp4")
+//	if err != nil {
+//	    panic(err)
+//	}
 //
 // Or you can generate a fingerprint from a memory buffer:
 //
-//     b, _ := ioutil.ReadFile("/path/to/file.mp4")
+//	b, _ := ioutil.ReadFile("/path/to/file.mp4")
+//
+//	ft, err := pexae.FingerprintBuffer(b)
+//	if err != nil {
+//	    panic(err)
+//	}
 //
-//     ft, err := pexae.FingerprintBuffer(b)
-//     if err != nil {
-//         panic(err)
-//     }
+// If you only want to use certain types of fingerprinting, you can
+// specify that as well:
+//
+//	ft1, _ := client.FingerprintFileForTypes("/path/to/file.mp4", pexae.FingerprintTypeAudio)
+//	ft2, _ := client.FingerprintBufferForTypes(b, pexae.FingerprintTypeVideo|pexae.FingerprintTypeMelody)
```

Comment:
The formatter changed a lot of this GoDoc, but this is the part that is actually new.